WooCommerce/Classes/ViewRelated/Products/Variations/Add Attributes/AddAttributeOptionsViewModel.swift

```diff
@@ -36,7 +44,7 @@ final class AddAttributeOptionsViewModel {
 
         /// Indicates if the view model is syncing a global attribute options
         ///
-        var isSyncing: Bool = false
+        var isSyncing: SyncState = .idle
```

Comment:
Probably worth it to rename this variable to `syncState` and update the documentation!